internal/pkg/cli/task_run.go

```diff
@@ -437,22 +437,21 @@ func BuildTaskRunCmd() *cobra.Command {
 	}
 	cmd := &cobra.Command{
 		Use:   "run",
-		Short: "Run a one-off task",
-		Long:  `Run a one-off task with configurations such as cpu-units, memory, image, etc.`,
+		Short: "Run a one-off task on Amazon ECS.",
 		Example: `
-Run a task with default setting. You will be prompted to specify a task group name and an environment for the tasks to run in.
+Run a task using your local Dockerfile and minimal container sizes. 
+You will be prompted to specify a task group name and an environment for the tasks to run in.
```

Comment:
Can we just have this be an optional flag (that we don't prompt for)? Instead just use copilot-task-run family or something?

Reply:
It's not optional with the current implementation🧐 do you mean using `copilot-task-run` as the default group name?

Reply:
What do we currently do for applications?

Reply:
Yeah, I think I'm okay with prompting the user for a name for the task that they're running. Is there a reason why you preferred it to be optional with a default?

Reply:
Just because i can imagine folks won't care too much what their one off task is called. I could be wrong, but like what would the flow be if I want to go run a quick sql query.

Oh, I wonder how many customers there are today. Hmmm, my DB is in a private subnet. ugh.
I know, I'll just run a shell task!
```sh
 copilot task run ./Dockerfile-sql --follow
   -> Would you like to run this in an environment? [y]
   -> Which? [prod]
   -> What do you want to call this task group? i-dont-care-really
```

That's how I imagine the flow kind of going.

Reply:
Since the task definition, the stack, the ecr repository and cw log group are all named after the group name, I can see a user running a task for a shell, and another one off task for a database migration and another one off task for something completely unrelated. This could lead to multiple different task definitions being tied to the group name, and wouldn't that be confusing (especially if users do it accidentally)?

What do you guys think about prompting but defaulting with a i) unique group name or ii) static group name?

Reply:
I think prompting but defaulting with a static name sounds good! That way we don't create a bunch of stacks with crazy names.